src/content/docs/extensions/full-text-search.md

```diff
@@ -0,0 +1,201 @@
+
+```cypher
+CREATE NODE TABLE Book (ID SERIAL, abstract STRING, author STRING, title STRING, PRIMARY KEY (ID));
+CREATE (b:Book {abstract: 'An exploration of quantum mechanics.', author: 'Alice Johnson', title: 'The Quantum World'});
+CREATE (b:Book {abstract: 'A magic journey through time and space.', author: 'John Smith', title: 'Chronicles of the Universe'});
+CREATE (b:Book {abstract: 'An introduction to machine learning techniques.', author: 'Emma Brown', title: 'Learning Machines'});
+CREATE (b:Book {abstract: 'A deep dive into the history of ancient civilizations.', author: 'Michael Lee', title: 'Echoes of the Past'});
+CREATE (b:Book {abstract: 'A fantasy tale of dragons and magic.', author: 'Charlotte Harris', title: 'The Dragon\'s Call'});
+```
+
+In the following sections, we will build a full-text search index on the book table, and demonstrate how to search for books relevant to a keyword query.
+
+### Create FTS index
+
+Kuzu provides a function `CREATE_FTS_INDEX` to create the full-text search index on a table:
+
+```cypher
+CALL CREATE_FTS_INDEX('TABLE_NAME', 'INDEX_NAME', ['PROP1', 'PROP2', 'PROP3'...], OPTIONAL_PARAM1 := 'OPTIONAL_VAL1')
+```
+- `TABLE_NAME`: The name of the table to build FTS index.
+- `INDEX_NAME`: The name of the FTS index to create.
+- `PROPERTIES`: A list of properties in the table to build FTS index on. Full text search will only search the properties with FTS index built on.
+
+The following optional parameters are supported:
+
+- `stemmer`: The text normalization technique to use. Should be one of: `arabic`, `basque`, `catalan`, `danish`, `dutch`, `english`, `finnish`, `french`, `german`, `greek`, `hindi`, `hungarian`, `indonesian`, `irish`, `italian`, `lithuanian`, `nepali`, `norwegian`, `porter`, `portuguese`, `romanian`, `russian`, `serbian`, `spanish`, `swedish`, `tamil`, `turkish`, or `none` if no stemming is to be used. Defaults to `english`,
+which uses a Snowball stemmer.
+
+The example below shows how to create an FTS index on the book table with the `abstract`, `author` and `title` properties using the `porter` stemmer.
+
+:::caution[Note]
+Kùzu uses special syntax for optional parameters. Note how the `:=` operator is used to assign a value
+to an optional parameter in the example below.
+:::
+
+```cypher
+CALL CREATE_FTS_INDEX(
+    'Book',   // Table name
+    'book_index',   // Index name
+    ['abstract', 'author', 'title'],   // Properties to build FTS index on
+    stemmer := 'porter'   // Stemmer to use (optional)
+)
+```
+
+Depending on the size of the dataset, the index creation may take some time. Once the index creation is complete,
+the index will be ready to use for full-text search.
+
+### Query FTS index
+
+Kuzu provides a table function `QUERY_FTS_INDEX` to query the FTS index on a table:
+
+```cypher
+CALL QUERY_FTS_INDEX(
+    'TABLE_NAME',
+    'INDEX_NAME',
+    'QUERY',
+    OPTIONAL_PARAM1 := 'OPTIONAL_VAL1'...
+)
```

Comment:
Should we complete the statement by adding RETURN at the end?

Reply:
@acquamarin could you confirm this? I agree with Guodong, this query seems incomplete without the RETURN.

Reply:
Fixed the query in: https://github.com/kuzudb/kuzu-docs/pull/358/files